README.md

```diff
@@ -78,6 +78,230 @@ print(hex(d.rip))
 d.kill()
 ```
 
+## Interaction with the Process
+When libdebug spawns a process using `d.run()`, it returns an object that allows interaction with the process. \
+For clarity and simplicity, the APIs provided are similar to those offered by [pwntools](https://github.com/Gallopsled/pwntools).
+
+#### recv
+Receives at most numb bytes from the child process stdout.
+```
+Args:
+    numb (int, optional): number of bytes to receive. Defaults to None.
```

Comment:
We should clarify that with the default `recv(numb=None)` it actually receives up to 4096 bytes. I would expect it to receive just a single byte.